game/hud/README.md

```diff
@@ -18,17 +18,6 @@ yarn
 
 You will then be ready to develop the HUD Module.
 
-If you are also making changes to the `library` you will need to link it as follows.
```

Comment:
with this new workspace setup will we no longer have to link library builds?

Reply:
Correct, will remove this

Reply:
Thanks